### PR TITLE
Perception: add cs2d tracker.

### PR DESCRIPTION
**Pull request by chaii**

### CODE REVIEW

File: modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.cc [hunk 1]
```diff
@@ -0,0 +1,97 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (ycool) [inline modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.cc]: s/2017/2018/

File: modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.cc [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    }
+  }
+
+  return true;
+}
+
+bool CS2DAffinityTracker::UpdateTracked(const cv::Mat &img,
+                                         const std::vector<Detected> &detected,
+                                         std::vector<Tracked> *tracked) {
+  return true;
```
Comment (ycool) [inline modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.cc]: add // TODO

File: modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.h [hunk 3]
```diff
@@ -0,0 +1,52 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (ycool) [inline modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.h]: s/2017/2018/

File: modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.h [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  bool GetAffinityMatrix(
+      const cv::Mat &img, const std::vector<Tracked> &tracked,
+      const std::vector<Detected> &detected,
+      std::vector<std::vector<float>> *affinity_matrix) override;
+
+  bool UpdateTracked(const cv::Mat &img, const std::vector<Detected> &detected,
+                      std::vector<Tracked> *tracked) override;
+
+ private:
+  float sz_lim = 0.5f;
```
Comment (ycool) [inline modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.h]: s/sz_lim/sz_lim_/

File: modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.h [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  float pos_range = 1.5f;
```
Comment (ycool) [inline modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.h]: s/pos_range/pos_range_/
lint check is using google c++ style: https://google.github.io/styleguide/cppguide.html

File: modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.cc (hunk 2 above)
Comment (chaii) [reply modules/perception/obstacle/camera/tracker/cs2d/cs2d_affinity_tracker.cc]: This is a pure virtual method in the base class, bu this tracker won't have this method in the future. Putting this for a dummy implementation.

**Pull request merged**